json_rpc/clients/httpclient.nim

```diff
@@ -65,17 +65,38 @@ method call*(client: RpcHttpClient, name: string,
   let
     id = client.getNextId()
     reqBody = $rpcCallNode(name, params, id)
-    req = HttpClientRequestRef.post(client.httpSession,
-                                    client.httpAddress.get,
-                                    body = reqBody.toOpenArrayByte(0, reqBody.len - 1),
-                                    headers = headers)
-    res =
-      try:
-        await req.send()
-      except CancelledError as e:
-        raise e
-      except CatchableError as e:
-        raise (ref RpcPostError)(msg: "Failed to send POST Request with JSON-RPC.", parent: e)
+
+  var req: HttpClientRequestRef
+  var res: HttpClientResponseRef
+
+  defer:
```

Comment:
@cheatfate after `req.send`, is there anything else but `close` that you can do with a request?